source/fundamentals/crud/read-operations/text.txt

```diff
@@ -0,0 +1,284 @@
+===========
+Search Text
+===========
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you will learn how to run a **text search** in MongoDB.
+
+In MongoDB, you can use a text search to retrieve documents that
+contain a term or a phrase in a specified field. A term is a sequence of
+characters that excludes whitespace characters. A phrase is a sequence
+of terms that with any number of whitespace characters.
+
+Sample Documents
+~~~~~~~~~~~~~~~~
+
+In the following sections, you run text searches on the
+``fast_and_furious_movies`` collection. Each section uses ``collection``
+to refer to the ``MongoCollection`` instance of the
+``fast_and_furious_movies`` collection. 
+
+This collection contains the following documents that represent one of
+the several movies that are part of the Fast and Furious movie franchise
+and tags they are mostly associated with:
+
+.. code-block:: json
+
+   { "_id": 1, "title": "2 Fast 2 Furious ", "tags": ["undercover", "drug dealer"] } 
+   { "_id": 2, "title": "Fast 5", "tags": ["bank robbery", "full team"] }
+   { "_id": 3, "title": "Furious 7", "tags": ["emotional"] }
+   { "_id": 4, "title": "The Fate of the Furious", "tags": ["betrayal"] }
+
+Text Index
+----------
+
+You must create a **text index** before running a text search. A text
+index specifies the string or string array field for the query to run a
+text search on. A collection can only contain one text index.
+
+In the following examples, you run text searches on the ``title``
+field in the ``fast_and_furious_movies`` collection. To enable text
+searches on the ``title`` field, create a text index using the
+:ref:`Indexes <index-text-indexes>` builder with the following 
+snippet: 
+
+.. literalinclude:: /includes/fundamentals/code-snippets/SearchText.java
+   :language: java
+   :dedent:
+   :start-after: begin textIndex
+   :end-before: end textIndex
+
+.. note::
```

Comment:
I think specifying a text index on multiple fields should be in its own section and the note should only apply to identify that they need to drop the index. I think placing it in the note relates it only to indexes on multiple fields, but it's an operation we would want them to know for other use cases such as when they no longer wanted it or wanted to change it to a different field.
i